diff --git a/node_modules/tr46/index.js b/node_modules/tr46/index.js
index 9ba36d8..8903a65 100644
--- a/node_modules/tr46/index.js
+++ b/node_modules/tr46/index.js
@@ -1,6 +1,6 @@
 "use strict";
 
-const punycode = require("punycode");
+const punycode = require("punycode/");
 const regexes = require("./lib/regexes.js");
 const mappingTable = require("./lib/mappingTable.json");
 const { STATUS_MAPPING } = require("./lib/statusMapping.js");
